Sources/ScipioKit/Runner.swift

```diff
@@ -175,7 +180,8 @@ extension Runner {
                 extraFlags: ExtraFlags? = nil,
                 extraBuildParameters: [String: String]? = nil,
                 enableLibraryEvolution: Bool? = nil,
-                frameworkModuleMapGenerationPolicy: FrameworkModuleMapGenerationPolicy? = nil
+                frameworkModuleMapGenerationPolicy: FrameworkModuleMapGenerationPolicy? = nil,
+                environment: [String: String]? = nil
```

Comment:
No changes for existing Scipio clients are required. As I said, it's rather a knob for advanced use cases like ours.